test/IntegrationTests/GraphQLTests.cs

```diff
@@ -96,16 +96,30 @@ public async Task SubmitsTraces(string packageVersion, bool setDocument)
         }
     }
 
-    private static void Request(List<RequestInfo> requests, string method = "POST", string url = "/graphql", string? body = null)
+    private static void Request(List<RequestInfo> requests, byte id, string method = "POST", string url = "/graphql", string? body = null)
     {
         requests.Add(new RequestInfo
         {
+            Id = GetTraceIdHex(id),
```

Comment:
Instead of these functions why not: `id.ToString("x32")`?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2725/commits/7a193faec75db264bcce85fa7201f4344e83fe78